src/System.Diagnostics.DiagnosticSource/src/Configurations.props

```diff
@@ -2,6 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
+      net45-Windows_NT;
```

Comment:
We are in the process of bringing up build support for netfx versions I would highly suggest you file a follow-up item to get that configuration enabled for this library if it is a desired configuration you want to support in the package.